gh-api.el

```diff
@@ -158,6 +158,7 @@
       (let ((req (oref resp :-req)))
         (oset resp :data-received nil)
         (oset req :url next)
+        (oset req :query nil)
```

Comment:
so I guess *ideally* we'd parse the link, split it in url/query, and use those parts in the followup request.
Maybe it's easy enough that we can do that right now?